### DISCUSSION TITLE
Multilayer map - Feature to toggle between different layers

### DISCUSSION THREAD

**Discussion by Geethanjali11** [apache/superset#18433] [Ideas]
## Description

Is there a feature to toggle between different layers in a deck.gl multiple layer map once it is published in a dashboard?
For example, if a map has 3 layers (scatterplot, line map, arc map) , I want to enable only scatterplot. A toggle button to change the layers.

**Comment by junlincc** [comment]
@Geethanjali11 thanks for suggesting. 
from product standpoint this is almost a  _must have_, however, I don't think we can get to it anytime soon. 
community contribution is always welcome!

**Comment by virtualarchitectures** [comment]
Hi, For the visualisation of multiple map layers this feature is fundamental and absolutely essential.

**Comment by Yisel-Hernandez** [comment]
Good afternoon everyone.

What happens is that I am trying to filter a multi-layer graph.

Does anyone have an idea of ​​how I could do so that as the graphs that make up the multi-layer graph are filtered, it is also filtered with the changes of the other graphs?

The idea I had was in the same dashboard to add the graphs that make up the multi-layer graph, but only the graphs are filtered separately but it does not affect the multi-layer graph in any way.
